src/Features/Core/Portable/Common/TaggedText.cs

```diff
@@ -83,6 +83,18 @@ internal TaggedText(string tag, string text, TaggedTextStyle style, string navig
 
         public override string ToString()
             => Text;
+
+        /// <summary>
+        /// Clones the current instance of <see cref="TaggedText"/> to a new instance with a different text.
+        /// 
+        /// Needed for truncating Exception Texts that are longer than a certain amount of characters.
+        /// </summary>
+        /// <param name="text">The new text to use for the clone of this instance.</param>
+        /// <returns>A new instance of <see cref="TaggedText"/> that clones the current instance.</returns>
+        internal TaggedText WithText(string text)
```

Comment:
Is this used?

Reply:
It does not look like it is, either I forgot to use it or I forgot to remove it.